app/controllers/api_new/users_controller.rb

```diff
@@ -12,4 +12,9 @@ class ApiNew::UsersController < ApiNew::ApiController
 	def current
 		@user = current_user
 	end
+
+	# get /api_new/users/current_nonprofit/object_events
+	def current_nonprofit_object_events
```

Comment:
As mentioned before the, this needs to handle the situation where the current user doesn't have a nonprofit

Reply:
What do u think should happen in that case? An error, or just redirect to a different page